CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## v5.0.1
+## Changed
+- Rename all component `.scss` files to `.module.scss` to align with [Create React App's convention for CSS modules](https://facebook.github.io/create-react-app/docs/adding-a-css-modules-stylesheet). This is a purely internal change and does not affect public consumption of the controls.

```

Comment:
Does this mean we can remove the :global from our css library

Reply:
Yes, we can remove it once ux-renderer updates to the new react scripts